### ISSUE TITLE
Document and simplify installation of dependencies

### ISSUE THREAD

**Issue by jonrkarr** [SystemsBioinformatics/cbmpy#27]
The documentation of the installation could more clearly outline the requirements and how they can be installed. 

Better, would be to add NumPy and the optional dependencies to `setup.py`:

```python
setuptools.setup(
    ...
   install_requires=['numpy'],
    extras_requires={
        'sympy': ['sympy']
        'glpk': [
            'swiglpk',
        ],
        'sbml': [
            'python_libsbml',
            'lxml',
        ],
        'all': [
            'sympy',
            'swiglpk',
            'python_libsbml',
            'lxml',
        ]
    },
)
```

This will allow simple optional installation, e.g.,
```
pip install cbmpy[sbml]
```

**Comment by bgoli** [comment]
Many thanks. I am in the process of updating the installation requirements and documentation. To start with I have added requirements.txt and environment.yml for pip and conda and will integrate your suggestions as well.

**Comment by bgoli** [comment]
included in 0.8.1
